scripts/create_mock_data.py

```diff
@@ -1,11 +1,21 @@
 import random
 from factories.expense import ExpenseFactory
 from factories.house import HouseFactory
+from factories.user import UserFactory
 
 
-def run():
-    for _ in range(500):
-        house = HouseFactory()
+def run(*args):
+    if args:
+        number_to_create = int(args[0])
```

Comment:
Why do you need it to be configurable? Who's going to change it?

Reply:
The developer that us this script
It able him to decide how many mock entities he would like to create